src/socketHandler/gameHandler.js

```diff
@@ -20,32 +20,36 @@ const { getByWebsocket } = require('../services/user.service');
 const Card = require('../utils/card.model');
 
 const startGame = async function (socket, io) {
-  const user = await getByWebsocket(socket.id);
-  const lobby = await getCurrentLobby(user.uuid);
-
   try {
-    await updateLobbyStatus(lobby.lobbyid, 'RUNNING');
-  } catch (e) {
-    console.log(`Error while updating lobby status: ${e}`);
-  }
+    const user = await getByWebsocket(socket.id);
+    const lobby = await getCurrentLobby(user.uuid);
+
+    try {
+      await updateLobbyStatus(lobby.lobbyid, 'RUNNING');
+    } catch (err) {
+      console.log(`Error while updating lobby status: ${err.message}`);
+    }
 
-  createGame(lobby.lobbyid, lobby.players);
+    createGame(lobby.lobbyid, lobby.players);
 
-  addSubround(lobby.lobbyid);
+    addSubround(lobby.lobbyid);
 
-  const players = Object.keys(getPlayers(lobby.lobbyid));
-  const gameData = startRound(1, players.length, lobby.maxRounds);
-  io.to(lobby.lobbyid).emit('startGame', gameData);
+    const players = Object.keys(getPlayers(lobby.lobbyid));
+    const gameData = startRound(1, players.length, lobby.maxRounds);
+    io.to(lobby.lobbyid).emit('startGame', gameData);
+  } catch (err) {
+    console.log(`Error while starting game: ${err.message}`);
```

Comment:
This prevents the backend from crashing when an error occurs, but the question remains. Do we want to notify the user/front end that there was an error in the socket events?

Reply:
It is not likely that these errors occur, so I will leave it as is. As discussed, we can later send the errors to the user via the socket.